presto-orc/src/main/java/com/facebook/presto/orc/writer/DecimalColumnWriter.java

```diff
@@ -69,6 +69,8 @@
 
     private final List<ColumnStatistics> rowGroupColumnStatistics = new ArrayList<>();
 
+    private long columnStatisticsRetainedSizeInBytes = 0;
```

Comment:
Looks like the part that update the `columnStatisticsRetainedSizeInBytes` it missing?